MediaPicker/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="sdcard_title">SD Card Required</string>
+    <string name="sdcard_message">A mounted SD card is required to upload media</string>
```

Comment:
Related issue: [Decide how to handle localisation. Import translations. (#29)](https://github.com/wordpress-mobile/WordPress-MediaPicker-Android/issues/29)